src/core/stream/representation/get_buffer_status.ts

```diff
@@ -87,11 +87,11 @@ export default function getBufferStatus(
   playbackObserver : IReadOnlyPlaybackObserver<unknown>,
   fastSwitchThreshold : number | undefined,
   bufferGoal : number,
+  bufferSizeGoal : number,
```

Comment:
Why is `wantedBufferSize` renamed here?

For `bufferGoal`, it was to explicit the fact that it is technically a different value:
  - `wantedBufferAhead` is what the application wants through the API and is received through a `loadVideo` API call
  - `bufferGoal` is what is actually enforced based on other device limitations. and is created by the `AdaptationStream`

Though here, the `wantedBufferSize` seems untouched since it is received through the API